app/code/Magento/RemoteStorage/Driver/Cache/CacheInterface.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\RemoteStorage\Driver\Cache;
+
+use League\Flysystem\FilesystemReader;
+
+/**
+ * Remote storage cache interface.
+ */
+interface CacheInterface extends FilesystemReader
```

Comment:
I wouldn't introduce this extension point if that is not necessary

Reply:
Moved Reader interface to CacheInterface.